storage/client/src/main/java/com/azure/storage/queue/models/DequeuedMessage.java

```diff
@@ -14,7 +14,7 @@
  * on a Queue.
  */
 @JacksonXmlRootElement(localName = "QueueMessage")
-public final class DequeuedMessageItem {
+public final class DequeuedMessage {
```

Comment:
Is there ever a time a user would want to mutate or instantiate this class? To me it would be reasonable to make this an immutable class?

Reply:
It wouldn't make sense to be mutated or to instantiate it, being an immutable class would make sense.

@jianghaolu How hard would this be to change in the AutoREST codegen?